tests/bluetooth/tester/src/btp/btp_gap.h

```diff
@@ -138,6 +138,16 @@ struct btp_gap_stop_advertising_rp {
 #define BTP_GAP_DISCOVERY_FLAG_OWN_ID_ADDR	0x20
 #define BTP_GAP_DISCOVERY_FLAG_USE_FILTER_LIST	0x40
 
+#define BTP_GAP_SET_DISCOVERY_PARAMS 0x2f
+#define BTP_GAP_PHY_LE_1M				0x01
+#define BTP_GAP_PHY_LE_CODED			0x02
```

Comment:
The first two of the above seem to have the value differently aligned than everything else in this file. Unless it's the GitHub web UI messing things up), please fix this.